mkdocs.yml

```diff
@@ -62,7 +62,9 @@ nav:
       - Removing Striping Noise:           how-to-guides/image-processing/removing-striping-noise.md
       - General Utility with FX:           how-to-guides/image-processing/general-utility-with-fx.md
       - Bundle Adjustment in ISIS:         how-to-guides/image-processing/bundle-adjustment-in-isis.md
-    - ISIS Demos and Workshops:            how-to-guides/isis-demos-and-workshops.md
+    - ISIS Demos and Workshops:
+      - Demos and Workshops:               how-to-guides/demos/isis-demos-and-workshops.md
```

Comment:
I think these should go under some archive group under "getting started". I don't think they are fleshed out enough as is.

Reply:
moved